packages/@lwc/template-compiler/src/__tests__/fixtures/directive-lwc-if-else/directive-for-each/html-elements/expected.js

```diff
@@ -0,0 +1,42 @@
+import { registerTemplate } from "lwc";
+function tmpl($api, $cmp, $slotset, $ctx) {
+  const {
+    k: api_key,
+    t: api_text,
+    h: api_element,
+    fr: api_fragment,
+    i: api_iterator,
+  } = $api;
+  return api_iterator($cmp.items, function (item) {
```

Comment:
@jodarove Added tests for the use case you brought up. We also already have karma tests for the iteration use case but I added a few scenarios there too.